language/de-DE/tpl_cassiopeia.ini

```
@@ -4,14 +4,23 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
 
-TPL_CASSIOPEIA_XML_DESCRIPTION="Weiterentwicklung der Space-Themes (Solarflare in 1.0, Milkyway in 1.5 und Protostar in 3.0), Cassiopeia ist das Joomla 4 Frontend Template basierend auf Bootstrap 4."
 
 CASSIOPEIA="Cassiopeia Frontend Template"
 TPL_CASSIOPEIA_BACKTOTOP="Zurück nach oben"
 TPL_CASSIOPEIA_BACKTOTOP_LABEL="„Zurück nach oben“ Link"
+TPL_CASSIOPEIA_COLOR_NAME_ALTERNATIVE="Alternativ"
+TPL_CASSIOPEIA_COLOR_NAME_LABEL="Theme Farbe"
```

```suggestion
TPL_CASSIOPEIA_COLOR_NAME_LABEL="Theme Farbe"
```

Theme ? Teamplate? ggf nur "Farbe"?

---

Template

---

Template richtig - es betrifft die Template-Farbe, oder?